content/docs/6.1/new-feature/overview.md

```diff
@@ -0,0 +1,74 @@
+---
+title: What's New
+description: New features and improvements about TiKV 6.1
+menu:
+    "6.1":
+        weight: 1
+        identifier: What's New-v6.1
+---
+
+This document lists some significant features and improvements in **TiKV 6.1**.
+
+
+## Improve stability of TiKV flow control
+
+TiKV introduces **a new flow control mechanism to replace the previous RocksDB write stall mechanism**. Compared with the write stall mechanism, this new mechanism **reduces the impact on the stability of foreground write**.
+
+In specific, when the stress of RocksDB compaction accumulates, flow control is performed at the TiKV scheduler layer instead of the RocksDB layer, to avoid the following issues:
+
+- Raftstore is stuck, which is caused by RocksDB write stall.
+
+- Raft election times out, and the node leader is transferred as a result.
+
+**This new mechanism improves the flow control algorithm to mitigate QPS decrease when the write traffic is high**.
+
+[Click here](https://docs.pingcap.com/tidb/stable/tikv-configuration-file#storageflow-control) to see user guide.
+
+## Wtrite latency improvement
+
+**Reduce the write latency by separating I/O operations from Raftstore thread pool** (disabled by default). For more information about tuning, see [Tune TiKV Thread Pool Performance](https://docs.pingcap.com/tidb/stable/tune-tikv-thread-performance).
+
+## Support Raft Engine (experimental)
+
+Support using Raft Engine as the log storage engine in TiKV. Compared with RocksDB, Raft Engine can **reduce TiKV I/O write traffic by up to 40% and CPU usage by 10%, while improving foreground throughput by about 5% and reducing tail latency by 20%** under certain loads. **In addition, Raft Engine improves the efficiency of log recycling and fixes the issue of log accumulation in extreme conditions**.
+
+Raft Engine is still an experimental feature and is disabled by default. Note that the data format of Raft Engine in v5.4.0 is not compatible with previous versions. Before upgrading or downgrading the cluster, you need to make sure that Raft Engine on all TiKV nodes is disabled. It is recommended to use Raft Engine only in v5.4.0 or a later version.
```

Comment:
```suggestion
Since v6.1.0, TiDB uses Raft Engine as the default storage engine for logs. Compared with RocksDB, Raft Engine can reduce TiKV I/O write traffic by up to 40% and CPU usage by 10%, while improving foreground throughput by about 5% and reducing tail latency by 20% under certain loads.
```